Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core/Fitnet.Contracts.Core.csproj

```diff
@@ -6,7 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="ErrorOr" Version="2.0.1" />
-    <PackageReference Include="EvolutionaryArchitecture.Fitnet.Common.Core" Version="4.1.1" />
+    <PackageReference Include="EvolutionaryArchitecture.Fitnet.Common.Core" Version="4.1.0" />
```

Comment:
revert this change

Reply:
Why not 4.1.2?

Reply:
4.1.2 version has not been build yet. This will be build after merge

Reply:
I have to update this in next MR